src/sentry/nodestore/bigtable/backend.py

```diff
@@ -168,6 +194,29 @@ def set(self, id, data, ttl=None):
         )
         self.connection.mutate_rows([row])
 
+    def get_multi(self, id_list):
+        if len(id_list) == 1:
+            id = id_list[0]
+            return {id: self.get(id)}
+
+        return {
+            id: future.result()
+            for id, future in izip(
+                id_list,
+                (
+                    self.thread_pool.submit(self.get, id)
```

Comment:
You can use `self.thread_pool.map` if you'd like (you probably know that already)